addons/website_sale/controllers/cart.py

```diff
@@ -16,11 +16,15 @@ class Cart(PaymentPortal):
 
     @route(route='/shop/cart', type='http', auth='public', website=True, sitemap=False)
     def cart(self, id=None, access_token=None, revive='', **post):
-        """
-        Main cart management + abandoned cart revival
-        id: Abandoned cart SO id
-        access_token: Abandoned cart SO access token
-        revive: Revival method when abandoned cart. Can be 'merge' or 'squash'
+        """ Display the cart page.
+
+        This route is responsible for the main cart management and abandoned cart revival logic.
+
+        :param str id: The abandoned cart's id.
+        :param str access_token: The abandoned cart's access token.
+        :param str revive: The revival method for abandoned carts. Can be 'merge' or 'squash'.
```

Comment:
`revive_method` would even be better. I'm tempted to pass a boolean to params named after a verb.

Reply:
There is a PR to clean the cart from a design POV and I intend to clean further the route and the JS in this PR: https://github.com/odoo/odoo/pull/190720. Let's discuss architectural changes and implement them in this PR.
In the meantime, I've rename the variable to `revive_method`